plaidml/edsl/ffi.cc

```diff
@@ -334,6 +349,7 @@ plaidml_expr* plaidml_expr_pragma(  //
     llvm::StringMap<ast::VarNodePtr> attrs;
     for (size_t i = 0; i < nattrs; i++) {
       plaidml_attr* attr = raw_attrs[i];
+      checkAttribute(attr->value->node);
```

Comment:
I think we should allow any VarNode here, why is this needed?

Reply:
I see that in the builder, we say unsupported, but I think we should just implement it at some point. I think the public interface should actually support it.

Reply:
I didn't see any VarNode that would trigger it, but there is a default error message thrown in the builder if an unknown varnode is specified; I surfaced that because it could be, but if it is not worth checking every use for that/there is indeed no input that would trigger it, I can remove it.

Reply:
I think it's the string type that would make it fail. I just don't think we should do a `checkAttribute` at this level. It shouldn't be possible in the edsl to create an invalid attribute.